suppressible-error-prone/src/main/java/com/palantir/suppressibleerrorprone/VisitorStateModifications.java

```diff
@@ -44,10 +47,18 @@ public static Description interceptDescription(VisitorState visitorState, Descri
         TreePath pathToActualError =
                 TreePath.getPath(visitorState.getPath().getCompilationUnit(), description.position.getTree());
 
-        Tree firstSuppressibleParent = Stream.iterate(pathToActualError, TreePath::getParentPath)
+        Tree firstSuppressibleParent = Stream.iterate(
+                        pathToActualError, treePath -> treePath.getParentPath() != null, TreePath::getParentPath)
                 .dropWhile(path -> !suppressibleKind(path.getLeaf().getKind()))
                 .findFirst()
-                .orElseThrow(() -> new RuntimeException("Can't find anything we can suppress"))
+                .orElseThrow(() -> {
+                    return new RuntimeException("Can't find any source element on the TreePath to the error to "
+                            + "place a @SuppressWarnings on. This is a bug an with suppressible-error-prone. "
+                            + "The path to the error is:\n\n"
+                            + StreamSupport.stream(pathToActualError.spliterator(), false)
+                                    .map(tree -> tree.getKind().name() + "\n===========================\n" + tree)
+                                    .collect(Collectors.joining("\n\n")));
```

Comment:
text block? 🥺

Reply:
I thought maybe but I don't think it's worth the bad formatting that palantir-java-format does without having actual newlines in the middle of the text.

Reply:
i.e.

```
new RuntimeException("""
  Can't find any source element on the TreePath to the error to place a @SuppressWarnings on. This is a bug with suppressible-error-prone.

  The path to the error is:

  %s

  ===================

  %s
  """.formatted(StreamSupport.stream(...), tree);

potentially translated wrong but I think it works

Reply:
We should probably fix PJF for this case:

```java
"""
Can't find any source element on the TreePath to the error to \
place a @SuppressWarnings on. This is a bug an with suppressible-error-prone.
The path to the error is:


"""
        + StreamSupport.stream(pathToActualError.spliterator(), false)
                .map(tree ->
                        tree.getKind().name() + "\n===========================\n" + tree)
                .collect(Collectors.joining("\n\n")));
```

should be

```java
"""
Can't find any source element on the TreePath to the error to \
place a @SuppressWarnings on. This is a bug an with suppressible-error-prone.
The path to the error is:


""" + StreamSupport.stream(pathToActualError.spliterator(), false)
        .map(tree ->
                tree.getKind().name() + "\n===========================\n" + tree)
        .collect(Collectors.joining("\n\n")));
```

imo

Reply:
It also newlines + double indents `.formatted`.

Reply:
true, but I would just do `"""...""".formatted(StreamSupport.stream())` and/or extract that to a variable. It is quite a gnarly expression to tack on

Reply:
> It also newlines + double indents `.formatted`.

oof

Reply:
Eh, I've done the text block version so if we fix PJF it should get autoformatted to something nice.